src/components/legend/_legend_item.scss

```
@@ -16,7 +16,7 @@ $legendItemVerticalPadding: $echLegendRowGap / 2;
   }
 
   &:hover {
```

This should actually only get background color on `:focus`. Leaving the underline as the hover state.
```suggestion
  &:focus {
```

---

Unfortunately, at the moment we don't have accessible interactions on the legend and thus `focus` would do nothing. This will be revisited in https://github.com/elastic/elastic-charts/issues/580.

Are you ok with it being `hover` until then?